src/utils/works.ts

```diff
@@ -53,5 +54,137 @@ export const getWorkUrl = ({
     workUrl += `/chapters/${chapterId}`;
   }
 
-  return workUrl
+  return workUrl;
 };
+
+export const getWorkPage = async (workId: string) => {
+  return cheerio.load(
+    (
+      await axios.get<string>(`https://archiveofourown.org/works/${workId}`, {
+        // We set a cookie to bypass the Terms of Service agreement modal that appears when viewing works as a guest, which prevented some selectors from working. Appending ?view_adult=true to URLs doesn't work for chaptered works since that part gets cleared when those are automatically redirected.
+        headers: {
+          Cookie: "view_adult=true;",
+        },
+      })
+    ).data
+  ) as WorkPage;
+};
+
+export interface Work {
+  author: string;
+  title: string;
+  rating: string[];
+  warning: string[];
+  fandom: string[];
+  language: string;
+  published: string;
+  words: string;
+  chapters: string;
+  category: string[];
+  relationship: string[];
+  character: string[];
+  freeform: string[];
+  series: string;
+  collections: string;
+  updated: string;
+}
+
+// Start of methods that fetch required fields
+export const getWorkAuthor = ($workPage: WorkPage) => {
+  const author = [];
+  $workPage("h3.byline").each(function (i, element) {
+    author[i] = $workPage(element).text().trim();
+  });
+  return author.join();
+};
+
+export const getWorkTitle = ($workPage: WorkPage) => {
+  return $workPage("h2.title").text().trim();
+};
+
+export const getWorkRating = ($workPage: WorkPage) => {
+  const rating = [];
+  $workPage("dd.rating ul.commas li").each(function (i, element) {
+    rating[i] = $workPage(element).text().trim();
+  });
+  return rating;
+};
+
+export const getWorkWarning = ($workPage: WorkPage) => {
+  const warning = [];
+
+  $workPage("dd.warning ul.commas li").each(function (i, element) {
+    warning[i] = $workPage(element).text().trim();
+  });
+  return warning;
+};
+
+export const getWorkFandom = ($workPage: WorkPage) => {
+  const fandom = [];
+  $workPage("dd.fandom ul.commas li").each(function (i, element) {
+    fandom[i] = $workPage(element).text().trim();
+  });
+  return fandom;
+};
+
+export const getWorkLanguage = ($workPage: WorkPage) => {
+  return $workPage("dd.language").text().trim();
+};
+
+export const getWorkPublished = ($workPage: WorkPage) => {
+  return $workPage("dd.published").text().trim();
+};
+export const getWorkWords = ($workPage: WorkPage) => {
+  return $workPage("dd.words").text().trim();
+};
+export const getWorkChapters = ($workPage: WorkPage) => {
+  return $workPage("dd.chapters").text().trim();
+};
+// End of methods that fetch required fields
```

Comment:
I don't think splitting these sections with comments makes sense. If you want to communicate this, the function signature is the best way to do so.

For example, you can define:
```ts
export const getWorkChapters = ($workPage: WorkPage) : string => {
```
(note the `: string`, which denotes the return value)
vs
```ts
export const getWorkSeries = ($workPage: WorkPage) : string | null => {
```
(note the `: string | null`, which denotes that the return value might be null)

Of course then you will have to make sure you do return null.